browser-extension/mv3/src/rulesStorageService.ts

```diff
@@ -74,7 +74,7 @@ class RulesStorageService {
 
       const group = groups.find((group) => group.id === rule.groupId);
 
-      if (group.status === Status.ACTIVE) {
```

Comment:
This will mark the rules as disabled when the group_id doesn't exist actually. I don't know if it is the right choice as entering a non existing groupId is really common in publicApi

Reply:
Should we mark the rules with non-existing groupId as Active?